src/utils/windowUtils.ts

```diff
@@ -41,6 +41,30 @@ export module Utils {
       }
     }
 
+    /**
+     * To be used in the first line of a deprecated method
```

Comment:
What does `deprecated()` do?  The action of the function should be the first sentence.  This sentence should probably go second.